app/(candidate)/dashboard/contacts/[[...attr]]/page.js

```diff
@@ -35,7 +39,23 @@ const fetchContacts = async ({
   }
 }
 
-const fetchPerson = async (personId) => {}
+const fetchPerson = async (personId) => {
+  const response = await serverFetch(
+    apiRoutes.contacts.get,
+    { id: personId },
+    {
+      next: {
+        revalidate: 3600,
+      },
+    },
+  )
+  if (response.ok) {
+    return response.data
+  } else {
+    console.warn('Failed to fetch person', response)
+    return null
```

Comment:
Shouldn't this show a "not found" view to the user and respond to the browser w/ a 404? 🤔